.github/workflows/emulator.yml

```diff
@@ -0,0 +1,25 @@
+name: Emulator-CI
+
+on:
+  push:
+    paths:
+      - 'emulator/**'
+
+jobs:
+  Emulator:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Clone Scalardl-tool repository
+      uses: actions/checkout@v2
+
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: Build Emulator
```

Comment:
`build` will trigger tests as well, so why don't you separate the runs like below ?
```
- name: Build Emulator
  run: ./gradlew assemble

- name: Test Emulator
  run: ./gradlew test
```

Reply:
Thank you review comment.
Workflow modified based on review comments.